tests/suites/test_suite_psa_crypto.data

```diff
@@ -1162,10 +1192,14 @@ PSA sign: deterministic ECDSA SECP256R1 SHA-256, output buffer too small
 depends_on:MBEDTLS_PK_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED:MBEDTLS_SHA256_C:MBEDTLS_ECDSA_C
 sign_fail:PSA_KEY_TYPE_ECC_KEYPAIR(PSA_ECC_CURVE_SECP256R1):"ab45435712649cb30bbddac49197eebf2740ffc7f874d9244c3460f54f322d3a":PSA_ALG_DETERMINISTIC_ECDSA( PSA_ALG_SHA_256 ):"9ac4335b469bbd791439248504dd0d49c71349a295fee5a1c68507f45a9e1c7b":63:PSA_ERROR_BUFFER_TOO_SMALL
 
-PSA sign: deterministic ECDSA SECP256R1, invalid hash
-depends_on:MBEDTLS_PK_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED:MBEDTLS_ECDSA_DETERMINISTIC:MBEDTLS_SHA256_C
+PSA sign: deterministic ECDSA SECP256R1, invalid hash (0)
```

Comment:
What's the `(0)` mean?

Reply:
It means `0` is used where a hash algorithm is expected.

Reply:
OK, this was clarifying the test that was already written to use `0` as hash. It has nothing to do with the update to the dependencies or to support wildcards, really.

Reply:
I changed the description of the text to make it unique, now that there is a second test case with an invalid hash. There's still a bit of room so I changed “invalid hash” to “invalid hash algorithm”.